ui/pages/swaps/prepare-swap-page/review-quote.test.js

```diff
@@ -93,7 +95,8 @@ describe('ReviewQuote', () => {
     const { getByText } = renderWithProvider(<ReviewQuote {...props} />, store);
     expect(getByText('New quotes in')).toBeInTheDocument();
     expect(getByText('Quote rate')).toBeInTheDocument();
-    expect(getByText('MetaMask fee')).toBeInTheDocument();
+    expect(getByText('Includes a 1% MetaMask fee –')).toBeInTheDocument();
+    expect(getByText('view all quotes')).toBeInTheDocument();
```

Comment:
Does this constitute a change in our fee or just moving it around?

Reply:
No change in the fee logic, just making it clearer that our MM fee is included in quotes already.